parsing/dialects/dialect1/parserImpls.ftl

```diff
@@ -2540,6 +2536,8 @@ SqlCall RangeNStartEnd() :
         <EACH>
         eachSizeLiteral = Literal()
     ]
-    { return new SqlRangeNStartEnd(getPos(), startLiteral, endLiteral
-        , eachSizeLiteral, startAsterisk, endAsterisk); }
+    {
+        return new SqlRangeNStartEnd(getPos(), startLiteral, endLiteral,
+        eachSizeLiteral, false, endAsterisk);
```

Comment:
nit: This line should be indented similar to the one on line 2514 in this file.

Reply:
Fixed in https://github.com/googleinterns/calcite/pull/212/commits/1c21c894edefe12eebf1715a79459f36994e826a